solana/Makefile

```diff
@@ -1,9 +1,9 @@
 .DEFAULT_GOAL = build
 .PHONY: build test unit-test integration-test clean
 
-ifndef (NETWORK)
-  $(info No NETWORK specified, defaulting to local cluster (devnet))
-  NETWORK = devnet
+ifndef NETWORK
+	$(info No NETWORK specified, defaulting to local cluster (devnet))
+	NETWORK = devnet
```

Comment:
I think this is breaking makefile parsing

replacing tabs with spaces seems to work
```
ben@Bens-Mac-mini solana % make unit-test
Makefile:20: *** recipe commences before first target.  Stop.
```